api/client/inspect.go

```diff
@@ -13,9 +13,7 @@ import (
 )
 
 // CmdInspect displays low-level information on one or more containers or images.
-//
```

Comment:
As the other client *.go files are, this line should be left to have the same format